### PR TITLE
Merge upstream/master into units-staging

### PR DESCRIPTION
**Pull request by younies**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
- [ ] Updated PR title and link in previous line to include Issue number
- [ ] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

File: icu4c/source/test/intltest/measfmttest.cpp [hunk 1]
```diff
@@ -81,10 +81,11 @@ class MeasureFormatTest : public IntlTest {
     void TestNumericTimeSomeSpecialFormats();
     void TestIdentifiers();
     void TestInvalidIdentifiers();
-    void TestParseToBuiltIn();
+    void TestKilogramIdentifier();
     void TestCompoundUnitOperations();
```
Comment (younies) [inline icu4c/source/test/intltest/measfmttest.cpp:85]: @hugovdm ... do we need this test case ?

Comment (hugovdm) [reply icu4c/source/test/intltest/measfmttest.cpp:85]: What are you trying to do? I merged upstream/master into units-staging in commit 7fee83b1af1bec25280754ea0d0884a4f510e4f7 (if I copy/pasted correctly). It looks like you're trying to merge upstream/master again? I meant you may want to merge units-staging into your java branch.

If I'm seeing correctly what you're doing (I'm on my phone making this a bit harder), you're merging upstream into units-staging, and you seem to be asking whether you can throw away unit tests as part of a merge? (I might be missing something?)

Comment (hugovdm) [reply icu4c/source/test/intltest/measfmttest.cpp:85]: I can take a closer look when I'm home at my laptop again.

Comment (hugovdm) [reply icu4c/source/test/intltest/measfmttest.cpp:85]: A
ha, I see: a merge picking up more recent changes:
- 35eae09a7c7743ab5911c1934e6e5fda1b9340f1 added TestKilogramIdentifier(); to ensure we are handling the kilogram case correctly.
- c2ae5b48efa996c779e180febcdd78c2792940c8 added TestParseToBuiltIn(); to prove that various complex identifiers do simplify to built-in units properly.

When merging branches, don't throw away code: if you think the test is not necessary, a separate pull request make sense. In this case, it's a very straight-forward conflict, two unit tests that were just placed in the same place (between the same two tests). When dealing with a merge conflict like this, simply keep both.

**Comment by hugovdm** [comment]
I think this branch can be deleted. (I may delete it in a few days if it's still around.)

**Review by hugovdm — COMMENTED** [review]
(This was closed in favour of https://github.com/icu-units/icu/pull/74)

**Pull request closed without merging**